src/main/java/com/google/api/codegen/transformer/csharp/CSharpStandaloneSampleTransformer.java

```diff
@@ -115,6 +118,21 @@ protected ClientMethodType fromCallingForm(CallingForm callingForm) {
     return CALLING_FORM_CLIENT_METHOD_TYPE_MAP.get(callingForm);
   }
 
+  @Override
+  protected SampleEntryPointView generateSampleEntryPoint(InterfaceContext interfaceContext) {
+    SampleEntryPointView.Builder entryPoint = SampleEntryPointView.newBuilder();
+    String sampleDirectory =
+        pathMapper.getOutputPath(
+            interfaceContext.getInterfaceModel().getFullName(),
+            interfaceContext.getProductConfig());
+    String outputPath =
+        new StringBuilder().append(sampleDirectory).append("/Program.cs").toString();
```

Comment:
Here, too, `Program` strikes me as too generic a name. How about `Runner`. @jskeet , what do you think?

Reply:
`Program` is a commonly used name for entry point class in C#.